src/xfacthd/depextract/html/Html.java

```diff
@@ -25,7 +25,16 @@ public static void element(HtmlWriter writer, String type, String attribs, Strin
         element(writer, type, attribs, contentWriter -> contentWriter.print(content));
     }
 
-    public static void element(HtmlWriter writer, String type, String attribs, Consumer<HtmlWriter> contentWriter)
+    public static void element(HtmlWriter writer, String type, String attribs, String content, boolean needsClosingTag)
+    {
+        element(writer, type, attribs, contentWriter -> contentWriter.print(content), needsClosingTag);
+    }
+
+    public static void element(HtmlWriter writer, String type, String attribs, Consumer<HtmlWriter> contentWriter) {
+        element(writer, type, attribs, contentWriter, true);
+    }
+
+    public static void element(HtmlWriter writer, String type, String attribs, Consumer<HtmlWriter> contentWriter, boolean needsClosingTag)
```

Comment:
I'd prefer an `elementNoBody()` method over the closing tag flag since an element without a body doesn't need the content writer